ru/frontend.php

```diff
@@ -99,4 +99,46 @@
     'settingsChanged' => 'Настройки были изменены во время игры. Повтор не будет сохранён.',
     'segment' => 'Сегмент',
     'duration' => 'Продолжительность',
+	
+	//New strings from July 29th -mLurker
+	'signingIn' => 'Регистрация',
+	'loginFail' => 'Не удалось войти!',
+	'mapLoading' => 'Загрузка карты...',
+	'oldVer2' => 'У Вас всё ещё устарешвая версия! Нажмите {key}, чтобы перезагрузить клиент.',
+	'roomFull' => 'Эта комната заполнена.',
+	'lobbyInfo' => 'Этот чат соединён с {discord} сервером.',
+	'newHost' => 'Вы теперь хозяин этой комнаты.',
+	'badRoom' => 'Эта комната исчезла! Перенаправление в Default room...',
+	'stngsChanged' => 'Настройки изменены хозяином',
+	'stngsCustom' => 'Польз. настройки',
+	'aSpec' => 'Вы уже наблюдаете!',
+	'aPlay' => 'Вы уже играете!',
+	'repFail' => 'Повтор не удалось сохранить.',
+	'repInChat' => 'Файл повтора выложен в чат. Скопируйте текст в окне и сохраните в текстовом файле.',
+	'repTxtInfo' => 'Этот файл можно потом открыть в проигрывателе на сайте или, возможно, переопубликовать на сайте.',
+	'newPB' => 'ВАШ НОВЫЙ РЕКОРД',
+	'firstPB' => 'Эта была Ваша первая игра. Поставьте ещё один рекорд, чтобы отметить ваши улучшения.',
+	'infoPB' => 'Ваш предыдущий рекорд: {prevPB}, поставленный {prevAgo}. Ваше улучшение: {PBdiff}.',
+	'daysAgo' => 'дней назад',
+	'raceFin' => 'ГОНКА ОКОНЧЕНА',
+	'raceFinInfo' => 'Вы можете закончить, но следующий раунд может начаться в любой момент.',
+	'oops' => 'Ой!',
+	'chatNA' => 'Общий чат не доступен гостям и пользователям с менее, чем {chReq} часов игры.',
+	'leMore' => 'Узнать подробности',
+	'connLimit' => 'Максимальный лимит открытых соединении достигнут для этого IP. Если Вам требуется увеличить лимит, свяжитесь с нами в Discord сервере',
+	'idleDC' => 'Вас отсоединили из-за бездействия! Места для наблюдателей заполнены.',
+	'RLreach' => 'Лимит соединении достигнут.',
+	'ban1' => 'Доступ к играм против игроков запрещён. Вы всё ещё можете играть в другие режимы.',
+	'ban2' => 'Этот пользователь ЗАБАНЕН. Войдите ещё раз для подробностей.', //Wrote "This user is BANNED.(...)" for brevity. -mLurker
```

Comment:
This message shows only to the banned user, right when they get banned.
Maybe better alternative would be `Your account is BANNED`